### PR TITLE
React router branch

### PR DESCRIPTION
**Pull request by 19aliaksandr90**

### CODE REVIEW

File: src/components/FavoritesMovieContainer.jsx [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
-import FavoritesMovieCard from "./FavoritesMovieCard";
-import styles from "../styles/favoritesMovies.module.css";
+import styles from "../styles/favoritesMovie.module.css";
 
-const FavoritiesMovieContainer = props => {
-	const { fullInfo } = props;
+import FavoritesMovie from "./FavoritesMovie";
+
+const FavoritesMovieContainer = props => {
+	const { favoritesMovie } = props;
```
Comment (yamixail) [inline src/components/FavoritesMovieContainer.jsx]: `favoriteMovies`?

File: src/components/ShortDescription.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import styles from "../styles/movieCard.module.css";
+
+const ShortDescriptionVideo = ({ title }) => {
+	return (
+		<Fragment>
+			<div className={`${styles.titleVideo}`}>
+				<span>
+					<b>{title}</b>
+				</span>
```
Comment (yamixail) [inline src/components/ShortDescription.jsx]: What is the purpose of using `span` here?

Comment (19aliaksandr90) [reply src/components/ShortDescription.jsx]: Fix.

File: src/components/FavoritesMovieContainer.jsx (hunk 1 above)
Comment (19aliaksandr90) [reply src/components/FavoritesMovieContainer.jsx]: The great and mighty Russian language. =) . Fixed

File: src/App.jsx [hunk 3]
```diff
@@ -137,16 +140,53 @@ class App extends React.Component {
...
 		return (
 			<div className='app'>
-				<Header />
-				<MovieContainer movies={movies.Search} />
-				<FavoritesMovieContainer fullInfo={fullDesc} />
+				<Router>
+					<Header />
+					<Switch>
+						<Route exact path='/'>
+							<MovieContainer movies={movies.Search} />
```
[the remark this answers is not in the document]
Comment (19aliaksandr90) [reply src/App.jsx]: Reasonable comment. Fixed

**Pull request merged**